src/geometamaker/cli.py

```diff
@@ -0,0 +1,155 @@
+import logging
+import os
+import sys
+
+import click
+from pydantic import ValidationError
+
+import geometamaker
+
+root_logger = logging.getLogger()
+root_logger.setLevel(logging.DEBUG)
+handler = logging.StreamHandler(sys.stdout)
+formatter = logging.Formatter(
+    fmt='%(asctime)s %(name)-18s %(levelname)-8s %(message)s',
+    datefmt='%m/%d/%Y %H:%M:%S ')
+handler.setFormatter(formatter)
```

Comment:
Super minor, but might I suggest using the all-caps reference names to clarify that these are module-level variables?

Reply:
Definitely, thank you!